docs/magento2/search-engine.mdx

```diff
@@ -405,8 +414,103 @@ Then, you need to define the API url in your environment variables:
 FRONT_COMMERCE_ATTRAQT_RECOMMENDATION_API_URL=http://api.early-birds.io/widget
 ```
 
-After this step, you should be able to use this loader in your modules to fetch
-Attraqt recommendations for your widgets.
+#### Using Attraqt Orchestrator Chrome extension
+
+In order to use the
+[Attraqt Orchestrator extension for Google Chrome](https://chrome.google.com/webstore/detail/attraqt-experience-orches/icognleckonecpphokboemgcgcohcepi),
+Front-Commerce exposes a GraphQL interface and provides a front-end component
+which does the needed setup. In this example, we'll make a module
+`SuggestedProduct` that fetches a recommendation from Attraqt via
+Front-Commerce's loader, and use it to initialize Attraqt Orchestrator
+
+:::note
+
+See [Extend the GraphQL schema](/docs/essentials/extend-the-graphql-schema) for
+instructions on how to add your own GraphQL module.
+
+:::
+
+You will first need to update your GraphQL schema to implement this interface
+and use it in a query:
+
+```graphql title="my-module/server/modules/SuggestedProducts/schema.gql"
+type ProductRecommendation implements AttraqtRecommendationResult {
+  id: ID!
+}
+
+extend type Query {
+  productRecommendation: ProductRecommendation
+}
+```
+
+Add a resolver for this query:
+
+```js title="my-module/server/modules/SuggestedProducts/resolvers.js"
+export default {
+  Query: {
+    productRecommendation: async (_, __, { loaders }) => {
+      const recommendation =
+        loaders.AttraqtRecommendations.loadRecommendationsForWidget(
+          "62f3c9d50cc94f235967672f"
+        );
+      const recommendedProductSku = recommendation.recommendations?.[0]?.id;
+      const product = loaders.Product.load(recommendedProductSku);
+      return {
+        ...recommendation,
+        product,
+      };
+    },
+  },
```

Comment:
That's where it becomes weird. In the real world, `id` might not actually be the SKU, all the objects inside `recommendations` entirely depends on what is configured in Attraqt (by Attraqt), so I hesitated a bit before putting it as an example in our docs.

Reply:
Yes, I understand. It might be clearer to indeed use another field name to make this clearer.

Maybe something like this:
```
// Note: the shape of recommendation.recommendations will vary
// depending on your data indexed in Attraqt. Adapt this example to your use case.
const skus = recommendation.recommendations.map(({ sku }) => sku);
```